package.json

```diff
@@ -4,6 +4,7 @@
   "version": "1.2.3",
   "description": "Lint markdown on the fly, using mdast-lint",
   "scripts": {
+    "lint": "mdast -u mdast-lint README.md LICENSE.md",
```

Comment:
Would have preferred wildcard, but changelog.md appears to be generated, so linting it would be a bit of a pain.